### ISSUE TITLE
will not parse my.cnf if any !include directives are present

### ISSUE THREAD

**Issue by ss8913** [hercules-team/augeas#113]
In 0.9.0, my my.cnf parses fine.  With 1.0.0 or newer, it does not.  If I take out the !include /sub/path.d/ directive, it works, but obviously I need that in there.  I saw in the changelog that something changed regarding !include directives, but it appears to have broken this functionality and is locking me into 0.9.0.

**Comment by lutter** [comment]
Is this still an issue ?

**Comment by ss8913** [comment]
Yes, I actually tested it yesterday with the latest augeas on one of my
Centos 6.6 hosts.  Still returns nothing when you do:

augtool print /files/etc/my.cnf

As long as the !include is present in the file.  Without !include, it works
fine.  Versions of augeas before 1.0.0 work fine either way.

**Comment by lutter** [comment]
The issue is actually not the include directive itself, but that Augeas parses that in a way that ends whatever section it appears in. The error happens on the line starting with `port` right after the `!includedir` line.

Not sure how to fix that though .. we'd probably have to accept that line as part of the `mysqld` section

**Comment by ss8913** [comment]
However it works in 0.9.0, it was OK then :)  Let me know if you want me to
test this for you, I have thousands of boxes that I can reproduce it on,
and I can send you one of the my.cnf files as well if I haven't already.
